core/controlplane/config/templates/cloud-config-worker

```diff
@@ -565,9 +580,16 @@ write_files:
              | base64 -d > $f
              mv -f $f ${encKey%.enc}
```

Comment:
If we go [that](https://github.com/kubernetes-incubator/kube-aws/pull/449/files#r108824744) way, the decrypted file must be moved to `/var/run/kubernetes/<a well-known name if any>` rather than `${encKey%.enc}`